test/Microsoft.AspNet.Mvc.Core.Test/ModelBinding/CompositeModelBinderTest.cs

```diff
@@ -24,14 +24,11 @@ public class CompositeModelBinderTest
                 ModelMetadata = new EmptyModelMetadataProvider().GetMetadataForType(typeof(int)),
                 ModelName = "someName",
                 ModelState = new ModelStateDictionary(),
+                OperationBindingContext = new OperationBindingContext(),
                 ValueProvider = new SimpleValueProvider
                 {
                     { "someName", "dummyValue" }
                 },
-                OperationBindingContext = new OperationBindingContext
-                {
-                    ValidatorProvider = GetValidatorProvider()
```

Comment:
Validators were still instantiated by this code even though they weren't used. I removed it.